src/server/plugins/reactHtml/index.jsx

```diff
@@ -236,6 +236,12 @@ export function getHead({
   `;
 }
 
+export function renderUseNativeIntlPolyfill(nonce) {
+  return process.env.ONE_CONFIG_USE_NATIVE_POLYFILL === 'true' ? `<script id="environment-variables" ${nonce}>
+        window.useNativePolyfill = 'true'
+       </script>` : '';
+}
```

Comment:
```suggestion
export function renderEnvironmentVariables(nonce) {
  return `<script id="environment-variables" ${nonce}>
        window.useNativePolyfill = ${process.env.ONE_CONFIG_USE_NATIVE_POLYFILL === 'true'}
       </script>`;
}
```

I think making this function a bit more general is better. It sets us up to add more env vars in the future

Reply:
I have made this function more generic in [d76bbe5](https://github.com/americanexpress/one-app/pull/1258/commits/d76bbe516d1724bdd802457057f5727466429a71)